modules/integration/tests-integration/tests/src/test/resources/automation_mapping.xsd

```diff
@@ -118,6 +119,15 @@
                         </xs:sequence>
                     </xs:complexType>
                 </xs:element>
+                <xs:element name="databasePath">
+                    <xs:complexType>
+                        <xs:sequence>
+                            <xs:element type="xs:string" name="H2DB"/>
+                            <xs:element type="xs:string" name="MySQL"/>
```

Comment:
Generalize XSD to facilitate to add any database type. In this case we can add only H2 and Mysql